online-shop/src/app/modules/shared/components/presentational/icon-button/icon-button.component.ts

```diff
@@ -0,0 +1,15 @@
+import { Component, Input, Output, EventEmitter} from '@angular/core';
+
+@Component({
+  standalone: true,
+  selector: 'app-icon-button',
+  templateUrl: './icon-button.component.html',
+  styleUrls: ['./icon-button.component.scss']
+})
+export class IconButtonComponent {
+  @Input() iconType !: String;
```

Comment:
You can use "string" here. Not "String": https://www.typescriptlang.org/docs/handbook/declaration-files/do-s-and-don-ts.html. Short version :D https://stackoverflow.com/questions/14727044/what-is-the-difference-between-types-string-and-string